checker/jdk/index/src/java/lang/String.java

```diff
@@ -162,7 +162,7 @@ public String() {
      * @param  original
      *         A {@code String}
      */
-    public @PolyMinLen @PolySameLen String(@PolyMinLen @PolySameLen String original) {
+    public /*@ PolyMinLen*/ @PolySameLen String(/*@ PolyMinLen*/ @PolySameLen String original) {
```

Comment:
Really? We use `/*@ AnnotationName*/` instead of `@AnnotationName` to comment out annotations throughout the Index Annotated JDK. If they're treated the same, I need to go through the whole Index JDK and change many things (especially the annotations on Strings, which I wrote but have since removed because we deferred String support).